packages/labs/analyzer/src/lib/javascript/functions.ts

```diff
@@ -23,6 +23,26 @@ import {getTypeForNode, getTypeForType} from '../types.js';
 import {parseJSDocDescription, parseNodeJSDocInfo} from './jsdoc.js';
 import {hasDefaultModifier, hasExportModifier} from '../utils.js';
 
+export const getDocNodeForFunctionLikeDeclaration = (
+  declaration: ts.FunctionLikeDeclaration,
+  analyzer: AnalyzerInterface
+) => {
+  if (ts.getJSDocTags(declaration).length !== 0) {
+    return declaration;
+  }
+
+  // Overloaded functions have mulitple declaration nodes. If there are no
+  // JSDoc tags on the provided declaration, use the first one that does have
+  // JSDoc tags for the purpose of extracting a description.
```

Comment:
TypeScript's behavior seems to be to choose the matched overload if there is one, or the first overload with jsdocs, but, surprisingly, _never_ show the impl docs. The impl basically doesn't exist.

https://www.typescriptlang.org/play?ts=5.1.0-dev.20230227#code/PQKhCgAIUh5A3ApgJwDYHsCGATSBGKEYcAMwFcA7AYwBcBLdCyTACgA8AuSAZxuTooBzAJRd46OtgDc4UBGhwkaLLgBMhYuWr1GzdlwpkAtgCMUoyOMky5hSAEkjAB1QbSlWgyatOPPgMFIAB9IQ1NzSABvKEgqRm50VEQAOgxBdmEZAF9wWWBIAAl0JQBaYpQeAAt0AHduSAAiBBQMHEhVBvBWPEzc2wVm5TaCaE0PHSYTfT9+IQsraXdtL0gp3zCzZHmJRf6YRxc3LU9dNa5eWcCQjYjoyFj4xJS0jOy+-KLS8uQq2vqmpStXB4TpTHoyJYnJhUaYXALbayQiaxaY3LZiHY2MB2A6uUZIlYw3xwoTBULGTbCKIxOIUBJJVLodJsXo5PKFb5lJS-OqhdCQABW3Gw6Co3HAMPB4CAA